framework/src/org/apache/cordova/CordovaWebViewImpl.java

```diff
@@ -553,11 +557,13 @@ public void onPageFinishedLoading(String url) {
                     public void run() {
                         try {
                             Thread.sleep(2000);
-                            cordova.getActivity().runOnUiThread(new Runnable() {
-                                public void run() {
-                                    pluginManager.postMessage("spinner", "stop");
-                                }
-                            });
+                            if (cordova.getActivity() != null) {
```

Comment:
If we do decide if logging debug statements is a good idea, then one should be also printed if this condition fails.